packages/client/hooks/useSnacksForNewMeetings.ts

```diff
@@ -32,7 +32,7 @@ const useSnacksForNewMeetings = (meetingsRef: any) => {
     const fiveSecsAgo = new Date(Date.now() - ms('5s'))
     const sortedMeetings = meetings
       .filter((meeting) => {
-        if (dismissedMeetingIds.has(meeting.id)) return false
+        if (dismissedMeetingIds.has(meeting?.id)) return false
```

Comment:
the next line will throw if meeting is undefined

Reply:
Updated!